S23/tanvipatel23/CodereviewAssignment.java

```diff
@@ -0,0 +1,30 @@
+new Locale(language)

```

Comment:
I would be great if we write them in a class, as new a key word to generate memory